link/src/main/java/com/stripe/android/link/ui/verification/VerificationScreen.kt

```diff
@@ -206,14 +154,6 @@ internal fun VerificationBody(
             )
         }
 
-        if (showChangeEmailMessage) {
```

Comment:
Are we not allowing email change?

Reply:
Nope. Since this will no longer be the full screen verify page, we don't need it. If the user wants to use a different email, they will click the close button and change it in the link inline enrollment field.